lib/module.php

```diff
@@ -438,6 +438,27 @@ public function save_hm_msgs() {
         }
     }
 
+    /**
```

Comment:
This function should be in modules/smtp/functions.php and follow the same structure as others